new_dawn_server/pusher/chat_service.py

```diff
@@ -0,0 +1,48 @@
+import pusher
+
+from new_dawn_server.settings import (
+	PUSHER_APP_ID,
+	PUSHER_APP_KEY,
+	PUSHER_APP_SECRET,
+	PUSHER_CLUSTER,
+)
+
+CHAT_EVENT = "chat_event"
+TEST_CHANNEL = "test_channel"
+MESSAGE = "message"
+USER_FROM_ID = "user_from_id"
+USER_TO_ID = "user_to_id"
+
+class ChatService:
+
+	def __init__(self, user_me_id=None, user_you_id=None):
+		self.user_me_id = user_me_id
+		self.user_you_id = user_you_id
+
+	def _build_channel_id(self):
```

Comment:
Can we make the channel id ignorant of numerical meaning?

Reply:
What do you mean by numerical meaning exactly? This is to make sure that each pair of users have an unique channel ID. In the future I will use a hash of this as channel ID.

Reply:
I see. I meant the greater than relationship, but I think hash will be good.